explorer/templates/explorer/play.html

```diff
@@ -14,6 +14,9 @@
     <div id="query_area" class="col-md-12">
         <h2>Playground</h2>
         <p>The playground is for experimenting and writing ad-hoc queries. By default, nothing you do here will be saved.</p>
+        {% if lag_exists %}
+          <div class="alert alert-warning">Warning! The queryset results might be outdated since some data were last updated around {{ replication_lag }}.</div>
```

Comment:
```suggestion
          <div class="alert alert-warning">Warning! The query results might be outdated since some data was last updated around {{ replication_lag }}.</div>
```

Reply:
I've updated it as mentioned here.